person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(name = 'Unknown', age = 0, parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    @age >= 18 || @parent_permission
+  end
+end
```

Comment:
- [OPTIONAL] Great job creating `Person`. It is important to adapt the best practice for arranging your code very well. It is recommended to set your Getter or Setter after class declaration, not after initialization. 
- [ ] It is required to set Setter for id and initialize it in your method
`attr_reader :id `

```
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```